CONTRIBUTING.rst

```diff
@@ -0,0 +1,75 @@
+Contributing to ZKAPAuthorizer
```

Comment:
While I definitely appreciate the benifits of sphix/restructuredtext of markdown, I originally wrote all the link in here with markdown syntax. I wonder if there is enough benefit of restructuredtext over markdown, given that both github and gitlab use markdown as the syntax for all the places you can write text in them.

Reply:
I don't have particularly strong feelings either way, and probably won't spend a lot of time thinking about it. This was mostly idle musing prompted by the combination of renaming this file to match its actual file-type and the my "muscle memory" for writing links in markdown format led to the incorrect formatting initially.

I *do* wonder a bit about the value of *sphix* (as opposed to rst vs md), given that we don't have the built docs available anywhere. Without that, I am much more like to go look at the source (either locally or in the VCS host), and neither github nor gitlab handle any of the sphinx extensions.